backend/src/main/java/ai/verta/modeldb/entities/ProjectEntity.java

```diff
@@ -73,8 +74,8 @@ public ProjectEntity(Project project) throws InvalidProtocolBufferException {
   @Column(name = "date_updated")
   private Long date_updated;
 
-  @Column(name = "project_visibility")
-  private Integer project_visibility;
+  @Transient
+  private VisibilityEnum.Visibility projectVisibility = VisibilityEnum.Visibility.PRIVATE;
```

Comment:
@coryverta we can not remove this directly because while migrating I can not get actual visibility from DB because the entity does not have that field. so in the first part, we have to migrate it and after successfully executing with new visibility logic we will remove it in the second round.

Reply:
```
 @Column(name = "project_visibility")
  private Integer project_visibility;
```
taking this revert since it is used on the migration